### ISSUE TITLE
Visual Studio Code is not picking up my Gulp Process

### ISSUE THREAD

**Issue by blachawk** [nickdodd79/vscode-gulptasks#33]
I'm not sure if this is a  bug or if I'm doing something wrong, but I have the following code within a `gulpfile.babel.js` file.  My Gulp Task extension for Visual Studio code is not recognizing the following...

```
import gulp from 'gulp';

export const serve = (done) => {
  console.log('living to serve')
  done();
}

const build = gulp.series(serve);

export default build;
```

What am I doing wrong?  I'm assuming I should see **serve** in my Gulp Task meta box for Visual Studio Code, but nothing shows up, even if I refresh.  It just says "no gulp files found"

**Comment by rogueco** [comment]
I'm having the same issue  - I am able to run the task through the command line without issue, but when trying to run via extension nothing is being picked up.

**Comment by MattouBatou** [comment]
TL;DR run `npm i -g gulp` to install gulp globally on your machine.

This is old now but anyone coming to the repo because of the same issue, ensure you have done the following:

- Install vscode-gulptasks extension in vscode.
- run `npm i -g gulp`
- Restart VSCode

VSCode should now add a "Gulp Tasks" dropdown to the bottom of your Explorer pane in VSCode.
